### PR TITLE
Fixes the math based on Jon's findings

### PR DESCRIPTION
**Pull request by katelovescode**
## 💅🏼 What issue does this fix?

Closes #714 with corrected math

## 🍂 Before You Submit

* [X] Did you write tests?
* [X] Did you run `bundle exec rspec` from the root?
* [X] Did you run `bundle exec rails rswag` from the root?
* [X] Did you run `bundle exec rubocop` from the root?

## 🧵 Steps to set up locally
N/A

### CODE REVIEW

File: spec/services/nebraska_hours_calculator_spec.rb [hunk 1]
```diff
@@ -56,17 +56,19 @@
           first_check_in = first_attendance_date.to_datetime + 8.hours
           first_check_out = first_check_in + 5.hours + 10.minutes
           second_check_in = second_attendance_date.to_datetime + 8.hours
+          second_check_out = second_check_in + 8.hours
```
Comment (katelovescode) [inline spec/services/nebraska_hours_calculator_spec.rb:59]: @JonErvin this addresses your point about the automatically created schedule, same in the next test

File: spec/services/nebraska_hours_calculator_spec.rb [hunk 2]
```diff
@@ -79,7 +81,7 @@
           second_check_out = second_check_in + 10.hours + 22.minutes
           create(:attendance, child_approval: child_approval, check_in: first_check_in, check_out: first_check_out)
           create(:attendance, child_approval: child_approval, check_in: second_check_in, check_out: second_check_out)
-          expect(described_class.new(child, first_attendance_date).call).to eq(5.25 + 0.25)
+          expect(described_class.new(child, first_attendance_date).call).to eq(5.25 + 0.5)
```
Comment (katelovescode) [inline spec/services/nebraska_hours_calculator_spec.rb:84]: @JonErvin You deffo found a bug in the formula itself, I don't know how I got that far assuming the 0.25 was right, so I changed the formula completely. 😳

File: app/services/nebraska_hours_calculator.rb [hunk 3]
```diff
@@ -20,13 +20,13 @@ def calculate_hours
...
+    adjusted_duration = if duration <= (5.hours + 45.minutes)
                           duration
-                        elsif duration > 10.hours && duration < (14.hours + 45.minutes)
+                        elsif duration > 10.hours && duration <= (14.hours + 45.minutes)
                           duration - 10.hours
                         else
                           0.minutes
                         end
-    (adjusted_duration.in_minutes / 60 * 4).round / 4.0
+    (adjusted_duration.in_minutes / 15.0).ceil * 15 / 60.0
```
Comment (katelovescode) [inline app/services/nebraska_hours_calculator.rb:30]: Me with math:

**Review by JonErvin — APPROVED** [review]
Looks great thanks Kate!

**Pull request merged**